.github/workflows/cpack.yml

```diff
@@ -123,17 +131,27 @@ jobs:
           ldd ./build/programs/ziti-edge-tunnel/ziti-edge-tunnel
           ./build/programs/ziti-edge-tunnel/ziti-edge-tunnel version --verbose
 
-      # - name: upload ziti-edge-tunnel binary
-      #   uses: actions/upload-artifact@v3
-      #   with:
-      #     name: ${{ matrix.distro.name }}-${{ matrix.distro.version }}-${{ matrix.arch }}-bin
-      #     path: |
-      #       ./build/programs/ziti-edge-tunnel/ziti-edge-tunnel
-      #     if-no-files-found: error
-
       - name: upload package artifact
         uses: actions/upload-artifact@v3
         with:
           name: ${{ matrix.distro.name }}-${{ matrix.distro.version }}-${{ matrix.arch }}-${{ matrix.distro.type }}
           path: ./build/ziti-edge-tunnel-*.${{ matrix.distro.type }}
           if-no-files-found: error
+
+      - name: rename package artifact for release
+        if: ${{ github.event_name == 'release' && startsWith(github.ref, 'refs/tags/v') }}
+        run: |
+          mv -v \
+            ./build/ziti-edge-tunnel-*.${{ matrix.distro.type }} \
+            ./build/ziti-edge-tunnel-${{ matrix.distro.name }}-${{ matrix.distro.version }}-${{ matrix.arch }}.${{ matrix.distro.type }}
+
+      - name: upload release artifacts
+        if: ${{ github.event_name == 'release' && startsWith(github.ref, 'refs/tags/v') }}
+        id: get_release
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: false
+          prerelease: false
+          fail_on_unmatched_files: true
+          files: |
+            ./build/ziti-edge-tunnel-${{ matrix.distro.name }}-${{ matrix.distro.version }}-${{ matrix.arch }}.${{ matrix.distro.type }}
```

Comment:
at the end of the day - how many things are going to be added to the 'release'?  are there going to be two more entries? one .deb and one .rpm?

[ziti-edge-tunnel-Darwin_x86_64.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Darwin_x86_64.zip)
[ziti-edge-tunnel-Linux_arm.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Linux_arm.zip)
[ziti-edge-tunnel-Linux_arm64.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Linux_arm64.zip)
[ziti-edge-tunnel-Linux_x86_64.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Linux_x86_64.zip)
[ziti-edge-tunnel-Windows_x86_64.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Windows_x86_64.zip)
[Source code (zip)](https://github.com/openziti/ziti-tunnel-sdk-c/archive/refs/tags/v0.18.17.zip)
[Source code (tar.gz)](https://github.com/openziti/ziti-tunnel-sdk-c/archive/refs/tags/v0.18.17.tar.gz)

Reply:
This is what it looks like with the additional Linux package artifacts in the release page: https://github.com/qrkourier/ziti-tunnel-sdk-c/releases/tag/v0.100.5

Reply:
i really don't want to add extra "thought" here for the user. let them download 'the one linux one' and call it a day and it should be 'the latest and greatest, get with the program if it doesn't work for you' imo.  when are we going to publish to some sort of repo (and doc the usage, i assume it'll be needed)

Reply:
The fulcrum of the argument against this unpopular PR seems to be that it introduces ambiguity to the decision point a user faces when they navigate to the GitHub Releases page for tSDK. Today, there's no such ambiguity. You simply choose the correct CPU architecture and OS.

The current plan is to move forward with the unstable distribution repo for YUM first, then APT. I'll close this PR unmerged.